pod/video/templates/channel/channel.html

```diff
@@ -198,40 +198,56 @@ <h2 class="card-title h4">
-                        $('footer.static-pod .hidden-pod').css('display', 'block');
-                        $('footer.static-pod').attr('style', 'height:auto;');
-                        $('footer.static-pod').removeClass('fixed-bottom');
+            onAfterPageLoad = function($items) {
+                document.querySelector('.infinite-loading').style.display = 'none';
+                let footer = document.querySelector('footer.static-pod');
+                if (!footer) return
+                footer.classList.add('small');
+                footer.classList.add('fixed-bottom');
+                footer.setAttribute('style', 'height:80px; overflow-y:auto');
+                document.querySelector('footer.static-pod .hidden-pod').style.display = 'none'
+                var docHeight = Math.max(
+                    document.body.scrollHeight,
+                    document.body.offsetHeight,
+                    document.clientHeight,
+                    document.scrollHeight,
+                    document.offsetHeight
+                  );
+                window.addEventListener("scroll", function () {
+                    if (window.innerHeight + window.scrollTop() === docHeight) {
+                      document.querySelector("footer.static-pod .hidden-pod").style.display =
+                        "block";
+                      footer.setAttribute("style", "height:auto;");
+                      footer.classList.remove("fixed-bottom");
                     }
-                });
+             });
             }
-        });
+
+
+        {% if videos.has_next %}
+            page = parseInt("{{videos.next_page_number}}")
+            next_page = true
+            var infinite = new InfiniteLoader(url, onBeforePageLoad, onAfterPageLoad, page = page);
+        {% endif %}
     </script>
     <script>
+
         var str = get_list(listTheme["channel_" + "{{channel.id}}"], 0, [], tag_type = "li", li_class = "list-group-item", attrs = '', add_link = true, current = "{{theme.slug}}", channel = "");
-        $("#ultreeview").html(str);
+        document.getElementById("ultreeview").innerHTML = str;
         setTimeout(function() {
-            const tree_item_info = $("#ultreeview li.list-group-item-info");
-            const tree_item = $("#ultreeview li")
-            if(!!tree_item_info.length && !!tree_item.length)
+            const tree_item_info = document.querySelector("#ultreeview li.list-group-item-info");
+            const tree_item = document.querySelectorAll("#ultreeview li")
+            if(tree_item_info && tree_item)
             {
-                const position = parseInt(tree_item_info.position().top) - parseInt(tree_item.first().position().top);
-                $("#ultreeview").scrollTop(position);
+                const position = parseInt(tree_item_info.offsetTop) - parseInt(tree_item[0].offsetTop);
+                document.getElementById("ultreeview").scrollTop(position);
```

Comment:
Lorsque j'affiche un thème de chaine sans vidéo, j'ai une erreur 
`TypeError: document.getElementById("ultreeview").scrollTop is not a function. (In 'document.getElementById("ultreeview").scrollTop(position)', 'document.getElementById("ultreeview").scrollTop' is 0)`

Reply:
je pense que c'est corrigé, c'est juste que scrolltop c'est une propiété non pas une méthode.